TrustSDK/Classes/Client/Types/UniversalAssetID.swift

```diff
@@ -0,0 +1,52 @@
+// Copyright Trust Wallet. All rights reserved.
+//
+// This file is part of TrustSDK. The full TrustSDK copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import Foundation
+import WalletCore
+
+/// https://developer.trustwallet.com/add_new_asset/universal_asset_id
+public struct UniversalAssetID: CustomStringConvertible, Equatable {
```

Comment:
Add tests for `UniversalAssetID`

Reply:
https://github.com/trustwallet/TrustSDK-iOS/blob/master/Example/Tests/AssetIDTests.swift